clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -0,0 +1,138 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 20 000 total QPS
```

Comment:
Ideally, the test should scale with the cluster size (as load test does). In such case we'll be able to run it along our CI/CD tests at 100, 500 and 5k node scale.

Reply:
Added linear scaling of QPS. Actually, we could get more QPS for smaller clusters, but this sets reasonable requirements and expectations what can be delivered.